cmd/kbn/config.go

```diff
@@ -280,11 +284,15 @@ func setgRPC(ctx *cli.Context, cfg *bootnodeConfig) {
 		cfg.GRPCHost = "127.0.0.1"
 		if ctx.GlobalIsSet(utils.GRPCListenAddrFlag.Name) {
 			cfg.GRPCHost = ctx.GlobalString(utils.GRPCListenAddrFlag.Name)
+		} else if ctx.GlobalIsSet(utils.LegacyGRPCListenAddrFlag.Name) {
+			cfg.GRPCHost = ctx.GlobalString(utils.LegacyGRPCListenAddrFlag.Name)
```

Comment:
It seems that the `getFlagName` job is also needed at here. Could you check it once more? And it seems same codes are exists in multiple files

Reply:
I reflected it. Thanks!